app/assets/javascripts/fae/form/inputs/_text.js

```diff
@@ -56,6 +57,38 @@ Fae.form.text = {
     $('.js-markdown-editor:not(.mde-enabled)').each(function () {
       var $this = $(this);
 
+      // set up translate button for markdown fields and add helper text
+      var translate_button = $this.siblings( '.js-translate-button' )
+      if (translate_button.length) {
+        var label = $this.siblings('.has_no_helper_text')
+        if (label.length) {
+          label.removeClass('has_no_helper_text')
+          var helper_class = jQuery('<h6>', {
+            class: 'helper_text',
+          })
+          jQuery('<span>', {
+            class: 'helper_text_text',
+            text: 'If machine translation is required, please click “Translate” button before applying any text formatting in markdown.'
+          }).appendTo(helper_class);
+
+          helper_class.appendTo(label)
+        }
+
+        var label_height = $this.siblings( 'label' ).height()
+        $this.siblings( '.js-translate-button' ).closest(".input").css({
```

Comment:
let's keep the CSS in stylesheets instead of within the JS. you can additional classes here via JS and then target those classes in a css file. `/app/assets/stylesheets/fae/modules/forms/_base.scss` would probably be a good place

Reply:
updated!